tests/e2e/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "../../tsconfig.base.json"
+}
```

Comment:
extending types with `puppeteer` like we do with `jest` and `jquery` doesn't expose the page object on global in tsconfig.base.json? Curious if we need to create the `global.d.ts`

Reply:
Added! Thanks!